File: application/src/main/java/whenyourcar/application/dto/user/like/UserLikeResponse.java
package whenyourcar.application.dto.user.like;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import org.springframework.data.domain.Page;
import whenyourcar.domain.query.car.SearchCarsQuery;

public class UserLikeResponse {
    @Builder
    @Getter
    @AllArgsConstructor
    public static class UserLikesResponseDto{
        Long userId;
        Page<SearchCarsQuery> searchCarsQueries;
    }
}

File: application/src/main/kotlin/howmuchcar/application/dto/user/UserLikeResponse.kt
package howmuchcar.application.dto.user

import howmuchcar.domain.query.car.SearchCarsQuery
import howmuchcar.domain.query.user.SearchCarLikeQuery
import org.springframework.data.domain.Page

data class UserLikeResponse (
    val userId: Long,
    val userLikeCars : Page<SearchCarLikeQuery>
){
}